frontend/web/components/pages/OrganisationSettingsPage.js

```diff
@@ -1,466 +0,0 @@
-import React, { Component } from 'react'
-import ConfirmRemoveOrganisation from 'components/modals/ConfirmRemoveOrganisation'
-import Payment from 'components/modals/Payment'
-import Button from 'components/base/forms/Button'
-import AdminAPIKeys from 'components/AdminAPIKeys'
-import Tabs from 'components/navigation/TabMenu/Tabs'
-import TabItem from 'components/navigation/TabMenu/TabItem'
-import JSONReference from 'components/JSONReference'
-import ConfigProvider from 'common/providers/ConfigProvider'
-import Constants from 'common/constants'
-import Icon from 'components/Icon'
-import _data from 'common/data/base/_data'
-import AccountStore from 'common/stores/account-store'
-import PageTitle from 'components/PageTitle'
-import SamlTab from 'components/SamlTab'
-import Setting from 'components/Setting'
-import AccountProvider from 'common/providers/AccountProvider'
-import LicensingTabContent from 'components/LicensingTabContent'
-import Utils from 'common/utils/utils'
-import AuditLogWebhooks from 'components/modals/AuditLogWebhooks'
-import MetadataPage from 'components/metadata/MetadataPage'
-import { withRouter } from 'react-router-dom'
-import SettingTitle from 'components/SettingTitle'
-const SettingsTab = {
-  'Billing': 'billing',
-  'CustomFields': 'custom-fields',
-  'General': 'general',
-  'Keys': 'keys',
-  'Licensing': 'licensing',
-  'SAML': 'saml',
-  'Usage': 'usage',
-  'Webhooks': 'webhooks',
-}
-
-const OrganisationSettingsPage = class extends Component {
-  static displayName = 'OrganisationSettingsPage'
-
-  constructor(props) {
-    super(props)
-    this.state = {
-      manageSubscriptionLoaded: true,
-      permissions: [],
-    }
-    if (!AccountStore.getOrganisation()) {
-      return
-    }
-    AppActions.getOrganisation(AccountStore.getOrganisation().id)
-
-    this.getOrganisationPermissions(AccountStore.getOrganisation().id)
-  }
-
-  componentDidMount = () => {
-    if (!AccountStore.getOrganisation()) {
-      return
-    }
-    API.trackPage(Constants.pages.ORGANISATION_SETTINGS)
-    $('body').trigger('click')
-  }
-
-  onSave = () => {
-    this.setState({ name: null }, () => {
-      toast('Saved organisation')
-    })
-  }
-
-  confirmRemove = (organisation, cb) => {
-    openModal(
-      'Delete Organisation',
-      <ConfirmRemoveOrganisation organisation={organisation} cb={cb} />,
-      'p-0',
-    )
-  }
-
-  onRemove = () => {
-    toast('Your organisation has been removed')
-    if (AccountStore.getOrganisation()) {
-      this.props.history.replace(Utils.getOrganisationHomePage())
-    } else {
-      this.props.history.replace('/create')
```

Comment:
Is there a reason why we changed the `/create` fallback? I haven't gone through this flow in a while but curious to have the rationale behind